apps/site-landing/src/components/RangeFieldAndScale/index.tsx

```diff
@@ -1,17 +1,17 @@
 import * as React from 'react';
 import { useState } from 'react';
-import styles from './styles.module.less';
 import { FormattedMessage, FormattedNumber, MessageDescriptor } from 'react-intl';
 import type { NumberFormatOptionsStyle } from '@formatjs/ecma402-abstract';
 
-import { Range, getTrackBackground } from "react-range";
-import { LessVars } from "@thecointech/site-semantic-theme/variables";
+import { Range, getTrackBackground } from 'react-range';
+import { LessVars } from '@thecointech/site-semantic-theme/variables';
+import styles from './styles.module.less';
```

Comment:
I like that you pay attention to the small things :-).  I always find it weird when `styles` isn't at the bottom of the import list too.  Is it silly?  Yes.  Does it matter?  No.  Do we do it anyway?  of course...